Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM jupyter/base-notebook:dd2087c75645
+FROM jupyter/base-notebook:latest
```

Comment:
you need to replicate these changes to the file [here](https://github.com/dotnet/interactive/blob/master/samples/my%20binder/Dockerfile)